aggregator/src/aggregator/aggregation_job_continue.rs

```diff
@@ -321,12 +322,7 @@ pub mod test_util {
                 .unwrap(),
             AggregationJobResp::MEDIA_TYPE
         );
-        let body_bytes = test_conn
-            .take_response_body()
-            .unwrap()
-            .into_bytes()
-            .await
-            .unwrap();
+        let body_bytes = take_response_body(&mut test_conn).await;
```

Comment:
100% of the places we use `take_response_body`, in this PR and in pre-existing code, we immediately decode the body bytes into some message via `get_decoded` (or, in a few places, a nearly-equivalent `decode` that should actually be a `get_decoded` to check that we didn't receive too much data).

WDYT about augmenting `take_response_body` to be generic on some `T: Decode` and return the decoded value, rather than the body bytes?

Reply:
Automerge beat me to it, but I'll explore this in another PR.